tests/ImageSharp.Tests/Processing/Normalization/HistogramEqualizationTests.cs

```diff
@@ -162,5 +163,40 @@ public void Issue984<TPixel>(TestImageProvider<TPixel> provider)
                 image.CompareToReferenceOutput(ValidatorComparer, provider);
             }
         }
+
+        [Theory]
+        [WithTestPatternImages(5120, 9234, PixelTypes.L16)]
+        public unsafe void Issue1640<TPixel>(TestImageProvider<TPixel> provider)
```

Comment:
How long does this run?
Instead of adding a 0.5MB reference image, I would prefer to create the test input dynamically, and would also make sure the test is isolated by using an custom `MemoryAllocator`.  Will post follow up.

Reply:
About 7 seconds on my machine. Wasn't concerned by the size since it's all LFS now but if you fancy working out a better approach go for it.

Reply:
Undecided if it's worth the extra time spent on minimizing the sizes or not. I'm a bit afraid that with growing test suite it will be a trouble in the future even with LFS, but maybe it's a false fear.

Reply:
Removed, using your test now.